### PR TITLE
Remove wire-maven-plugin

### PR DESCRIPTION
**Pull request by jrodbx**
RIP

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -79,15 +80,6 @@ buildscript {
           'gradle': "ru.vyarus:gradle-animalsniffer-plugin:${versions.animalSnifferGradle}",
           'annotations': "org.codehaus.mojo:animal-sniffer-annotations:${versions.animalSniffer}",
       ],
-      'maven': [
-          'core': "org.apache.maven:maven-core:${versions.maven}",
-          'plugin': [
-              'api': "org.apache.maven:maven-plugin-api:${versions.maven}",
-              'annotations': "org.apache.maven.plugin-tools:maven-plugin-annotations:${versions.mavenAnnotations}",
```
Comment (Egorand) [inline build.gradle]: Cleanup `versions` map?

File: wire-codegen-sample/build.gradle [hunk 2]
```diff
@@ -1,4 +1,3 @@
-apply plugin: 'com.liferay.maven.plugin.builder'
```
Comment (Egorand) [inline wire-codegen-sample/build.gradle]: We should rewrite `wire-codegen-sample` to use Gradle, now it won't generate code.

Comment (jrodbx) [reply wire-codegen-sample/build.gradle]: wanna tackle that in a follow-up?

Comment (Egorand) [reply wire-codegen-sample/build.gradle]: For sure!

**Pull request merged**